source/change-streams.rst

```diff
@@ -282,28 +305,36 @@ Driver API
      */
     collation: Optional<Document>;
 
+    /**
+     * The changeStream will only provides changes that occurred after the
+     * specified cluster timestamp. The default value is a cluster time obtained from
+     * the server before the changestream was created.
+     * @since 4.0
+     * @see https://docs.mongodb.com/manual/reference/method/db.runCommand/
+     */
+    startAtClusterTime: Optional<ClusterTime>;
```

Comment:
Don't add a `ClusterTime` type unless it's defined somewhere (all the other types in the options class are defined in the BSON spec).  Just call it `Document` and specify where an application should retrieve the value from.

Reply:
Done